commercial-partner.md

```diff
@@ -0,0 +1,52 @@
+# Commercial Partner
+
+**Note:** This document is a work in progress
+
+The Karmada commercial partner program offers Karmada support, consulting, professional services for organizations embarking on their Karmada journey.
+
+Karmada commercial partners are any organization that successfully leverages the Karmada project for their service offering. Such as:
+- Using Karmada to build service on a public or private cloud.
+- Repackaging Karmada as a core component of a service offering.
+- Offering Karmada technical support, consultation services, and training.
+- Others leverage Karmada in the manner it was intended to serve their customers.
+
+### Why become a partner?
+
+If your company provides professional services to support Karmada deployments, please consider applying to become a Karmada commercial partner.
+
+The main benefits of becoming a Karmada commercial partner are:
+* Placement on the first tab of https://karmada.io/
+* More community meeting exposure opportunities, such as the following meetings
+  * KubeCon.
+  * Online and offline meetings held by the Karmada community.
+  * Online and offline meetings participated by the Karmada community(such as KCD).
+* CNCF official media exposure opportunities, such as official Blog, Twitter, public accounts, etc.
+* Karmada security vulnerability disclosed in advance.
+* Karmada Community Supported Job Postings.
+* Community business opportunity delivery and connection.
+
+### How to become a partner?
+
+If your company is interested in becoming a Karmada commercial partner, please complete the following steps:
+
+1. Prepare the following information in advance
+   * Company official name.
+   * Company official website address(related to the Karmada commercial partner).
+   * Company Contact information email or github ID.
+   * Company logo (png format).
+2. In [community](https://github.com/karmada-io/community.git) repo, select the issue template of `Karmada support Partner Request` to create an issue and fill in the above information.
+
+Karmada maintainers will review these issues, and if the information is correct, a formal PR will be created. At this time, the applicant organization will officially join the Karmada Commercial Partner.
+
+### Karmada commercial partner exit mechanism
+
+There is currently no withdrawal mechanism, but the Karmada Community Governance Committee reserves the right to withdraw commercial partners.
```

Comment:
We don't have "Karmada Community Governance Committee" yet, so maybe keep Karmada maintainer team here.

Reply:
done